ironfish-rust-nodejs/src/lib.rs

```diff
@@ -15,6 +16,10 @@ pub mod nacl;
 pub mod rolling_filter;
 pub mod structs;
 
+fn to_napi_err(err: impl Display) -> napi::Error {
```

Comment:
This is a stop-gap to make it a little easier to convert to napi errors (which are effectively strings passed to the Javascript) until I can think of a cleaner way to deal with it